qwp_bench_test.go

```
@@ -0,0 +1,249 @@
+	s.globalSymbols["AAPL"] = 0
+	s.globalSymbolList = append(s.globalSymbolList, "AAPL")
+	s.batchMaxSymbolId = 0
+
+	iter := func() {
+		for r := 0; r < 10; r++ {
+			s.Table("t").
+				Symbol("sym", "AAPL").
+				Int64Column("qty", int64(100+r)).
+				Float64Column("price", 150.5+float64(r)).
+				StringColumn("note", "test").
+				At(ctx, ts.Add(time.Duration(r)*time.Microsecond))
```

golangci-lint (2.11.4)

[error] Error return value of `(github.com/questdb/go-questdb-client/v4.LineSender).At` is not checked

(errcheck)